README.md

```diff
@@ -4,3 +4,26 @@ Installers for MNE-Python for macOS, Windows, and Linux.
 
 ⚠️ Nothing to see here for now, please move along and check back again in a few
 days. ⚠️
+
+
+
+## Installation
+
+To install MNE-Python simply double click the installer.
```

Comment:
This is only the case for Windows and macOS. On Linux, users will have to run the `.sh` file in the shell via `sh xxx.sh` or `bash xxx.sh`